Dangerfile

```diff
@@ -13,8 +13,8 @@ This PR doesn't match our required code formatting standards, as enforced by pre
   })
 end
 
-locales_changes = git.modified_files.grep(/config\/locales/)
-has_non_en_locales_changes = locales_changes.grep_v(/config\/locales\/(client|server)\.en\.yml/).any?
+locales_changes = git.modified_files.grep(%r{config/locales})
+has_non_en_locales_changes = locales_changes.grep_v(%r{config/locales/(?:client|server)\.(?:en|en_US)\.yml}).any?
```

Comment:
We don't want `en_US` to be on Transifex, so it should get the same protection as `en`.